Fixes build failure against recent binutils:

/tmp/ghc23818_0/ghc23818_0.s: Assembler messages:
/tmp/ghc23818_0/ghc23818_0.s: Error: .size expression for PluginziPlziCommon_operators19_entry does not evaluate to a constant
/tmp/ghc23818_0/ghc23818_0.s: Error: .size expression for PluginziPlziCommon_operators139_entry does not evaluate to a constant
/tmp/ghc23818_0/ghc23818_0.s: Error: .size expression for PluginziPlziCommon_operators135_entry does not evaluate to a constant
/tmp/ghc23818_0/ghc23818_0.s: Error: .size expression for PluginziPlziCommon_operators69_entry does not evaluate to a constant
/tmp/ghc23818_0/ghc23818_0.s: Error: .size expression for PluginziPlziCommon_operators67_entry does not evaluate to a constant
diff --git a/Plugin/Pl/Common.hs b/Plugin/Pl/Common.hs
index 83c625c..29e6b04 100644
--- a/Plugin/Pl/Common.hs
+++ b/Plugin/Pl/Common.hs
@@ -1,5 +1,3 @@
-{-# OPTIONS -fvia-C #-}
-
 module Plugin.Pl.Common (
         Fixity(..), Expr(..), Pattern(..), Decl(..), TopLevel(..),
         bt, sizeExpr, mapTopLevel, getExpr,
diff --git a/dist/build/lambdabot/lambdabot-tmp/Plugin/Pl/Common.hi b/dist/build/lambdabot/lambdabot-tmp/Plugin/Pl/Common.hi
index 2bef562..48fbd52 100644
Binary files a/dist/build/lambdabot/lambdabot-tmp/Plugin/Pl/Common.hi and b/dist/build/lambdabot/lambdabot-tmp/Plugin/Pl/Common.hi differ
